src/plugins/condition/components/Condition.vue

```diff
@@ -23,7 +23,6 @@
 <template>
 <div v-if="isEditing"
      class="c-condition c-condition--edit js-condition-drag-wrapper"
```

Comment:
Are you just getting rid of all the current condition stuff to re-do it properly? Should we have a new issue for that?

Reply:
Yes. We can create a new issue for that.

Reply:
https://github.com/nasa/openmct/issues/2749